docs/readmes/howtos/troubleshooting/agw_unable_to_checkin.md

```diff
@@ -67,12 +68,12 @@ rootca_cert: /var/opt/magma/tmp/certs/rootCA.pem
     `AGW$ sudo tail -f /var/log/syslog | grep -i "magmad"`
 
 
-
-9. From Orchestrator, get all pods and find pod orc8r-controller-*
+9. From Orchestrator, get all pods and verify attempts from AGW are reaching Orc8r in nginx and look for any bootstrapping erors in the bootstrapper.
```

Comment:
Add an example

Reply:
@karthiksubraveti I Added more details and example in the command